### PR TITLE
Update Proxy "ownKeys" handler, duplicate keys for non-extensible targets test

### LINKED ISSUE #1164: Returning duplicates from the Proxy trap [[OwnPropertyKeys]] now throws
See https://github.com/tc39/ecma262/pull/833. This invalidates the 2017 misc test: Proxy "ownKeys" handler, duplicate keys for non-extensible targets.

### PR DESCRIPTION
**Pull request by chicoxyzzy**
Closes #1164

### CODE REVIEW

File: data-es2016plus.js [hunk 1]
```diff
@@ -2262,40 +2262,29 @@ exports.tests = [
...
-      chrome51: true,
-      chrome75: false,
-      safari10: true,
-      safari10_1: true,
-      safari13: false,
-      duktape2_0: true,
-      graalvm: true,
+      edge16: false,
+      chrome75: true,
+      firefox70: true,
```
Comment (zloirock) [inline data-es2016plus.js:2287]: FF57

**Review by zloirock — CHANGES_REQUESTED** [review]

**Review by zloirock — APPROVED** [review]

File: data-es2016plus.js (hunk 1 above)
Comment (chicoxyzzy) [reply data-es2016plus.js:2287]: thanks

**Comment by zloirock** [comment]
Seems it's also should be moved to ES2018 since the related PR was merged in August 2017.

**Comment by chicoxyzzy** [comment]
Good catch!

**Pull request merged**